### ISSUE TITLE
HTML in data-tip stopped working after upgrading from 3.8.0 to 3.8.4

### ISSUE THREAD

**Issue by abdulrahman-khankan** [wwayne/react-tooltip#446]
```javascript
<a
  data-tip={`<img src="https://upload.wikimedia.org/wikipedia/mediawiki/a/a9/Example.jpg"/>`}
  data-html={true}
  href={`https://upload.wikimedia.org/wikipedia/mediawiki/a/a9/Example.jpg`}
  target="_blank"
  style={{ textDecoration: "underline" }}
>
```
The above chunk used to work perfectly in 3.8.0. And now in 3.8.4 (and 3.9.0) it doesn't. Most likely due to the new sanitize feature

If I replace the HTML with just the link as a text. Then it works perfectly.
i.e. replace `<img src="https://WHATEVER_LINK"/>` with `data-tip={"https://WHATEVER_LINK"}` 

So the problem is only when passing HTML

**Comment by abdulrahman-khankan** [comment]
When inspecting the tooltip div
```html
<!-- 3.8.0 -->
<div class="__react_component_tooltip place-left type-dark assetMonitorTooltip" data-id="tooltip" style="left: 799px; top: 202px;">
  <img src="https://upload.wikimedia.org/wikipedia/mediawiki/a/a9/Example.jpg">
</div>
```

```html
<!-- 3.8.4 and 3.9.0 -->
<div class="__react_component_tooltip place-top type-dark assetMonitorTooltip" data-id="tooltip" style="left: 1080px; top: 177px;">

</div>
```

**Comment by abdulrahman-khankan** [comment]
This has been resolved in 3.9.2
It doesn't use sanitize-html anymore.

**Comment by aronhelser** [comment]
Thanks for following up!
